dokka-runners/dokka-gradle-plugin/src/main/kotlin/formats/DokkaFormatPlugin.kt

```diff
@@ -65,9 +65,9 @@ abstract class DokkaFormatPlugin(
         target.pluginManager.apply(DokkaBasePlugin::class)
 
         // apply the plugin that will autoconfigure Dokka to use the sources of a Kotlin project
-        target.pluginManager.apply(type = KotlinAdapter::class)
+        KotlinAdapter.applyTo(target)
+        AndroidAdapter.applyTo(target)
         target.pluginManager.apply(type = JavaAdapter::class)
```

Comment:
minor: it would be better if `JavaAdapter` followed the same apply convention as others